test/assert.js

```diff
@@ -182,6 +183,29 @@ test('.throws()', function (t) {
 	t.end();
 });
 
+test('.throws() becomes the thrown Error', function (t) {
```

Comment:
Would you like for me to change the subject of the Promise rejection while I'm at it? @novemberborn 

> '.throws() becomes the rejection reason of promise'
> '.throws() returns the rejection reason of promise'